index.bs

```diff
@@ -2344,6 +2363,42 @@ when invoked, must run these steps:
 
 </div>
 
+<div class=algorithm>
+
+The <dfn method for=IDBFactory>databases()</dfn> method,
+when invoked, must run these steps:
+
+1. Let |origin| be the [=/origin=] of the global scope used
+    to access this {{IDBFactory}}.
+
+1. If |origin| is an [=opaque origin=],
+    then return a new promise rejected with a "{{NotSupportedError}}" {{DOMException}}.
+
```

Comment:
The other `IDBFactory` methods (`open` and `deleteDatabase`) throw a `SecurityError` DOMException upon an opaque origin. Is this deviation here deliberate?

Reply:
Also, I see no indication, unlike the other `IDBFactory` methods, of what should happen if there are any errors in retrieving the info... If this would occur as a rejected Promise, does the error format need to be documented?